lib/rocketfuel_api/service.rb

```diff
@@ -42,8 +43,6 @@ def get_all(params = {})
   end
 
   def create(route_params = {}, body = {})
```

Comment:
Should't we remove the `create` and `update` methods?

Reply:
Why? Shouldn't an API client have them?

Reply:
Hum... before all services were setting read_only = true, so none could run `create` or `update`. So I thought we don't need them. But for some services create/update really doesn't make sense, for example for reporting. Should we add read_only logic back and set read_only = false for the ones that can create/updage?

Reply:
The `read_only` part was something we copied over from the `appnexusapi` gem and deemed not necessary. [The API](https://api.rocketfuel.com/doc/2016/index.html) has endpoints to create and update things. It's not that well structured, but if something doesn't have a create or update endpoint, the API will tell you anyway. So, I'd just leave this in there.